scenarios/kitsune_continuous_flow/src/main.rs

```diff
@@ -0,0 +1,39 @@
+use kitsune_wind_tunnel_runner::prelude::*;
+use std::time::Duration;
+
+fn agent_setup(ctx: &mut AgentContext<KitsuneRunnerContext, KitsuneAgentContext>) -> HookResult {
+    create_chatter(ctx)?;
+    join_chatter(ctx)?;
+    ctx.runner_context().executor().execute_in_place(async {
+        tokio::time::sleep(Duration::from_secs(15)).await;
+        Ok(())
+    })
+}
+
+fn behavior(
+    ctx: &mut AgentContext<KitsuneRunnerContext, KitsuneAgentContext>,
+) -> anyhow::Result<()> {
+    let message = format!(
+        "message_{}",
+        std::time::UNIX_EPOCH
+            .elapsed()
+            .expect("time went backwards")
+            .as_millis()
+    );
+    say(ctx, &message)?;
+    ctx.runner_context().executor().execute_in_place(async {
+        tokio::time::sleep(Duration::from_secs(20)).await;
```

Comment:
Maybe make this a configuration option for the test through an env var?

Reply:
Although, we're doing that stuff on a later ticket right?

Reply:
Yes, exactly. I haven't spent a lot of time on refining the scenario and its parameters etc., because that's part of the next issue.